nanozap/service/InvoiceService.swift

```diff
@@ -45,7 +45,7 @@ class InvoiceService {
         }
     }
 
-    public func payInvoice(invoice: Invoice) throws -> Bool {
+    public func payInvoice(invoice: DecodedInvoice) throws -> Bool {
```

Comment:
Vurder å lage en egen type, PayableInvoice som holder nøyaktig feltene som trengs for å gjennomføre en betaling

Reply:
Evt PayInvoiceRequest

Reply:
Evt PayInvoiceRequest